app/models/editorial_remark.rb

```diff
@@ -3,4 +3,16 @@ class EditorialRemark < ApplicationRecord
   belongs_to :author, class_name: "User"
 
   validates :edition, :body, :author, presence: true
+
+  def is_for_newer_edition?(edition)
```

Comment:
Seems a shame to have to duplicate this - did you consider any alternatives (e.g. extract into a module?)

I don't feel strongly about it though, agree that duplication is better than the wrong abstraction! And at least the code is simple.

Reply:
I agree with Stu's duplication choice - the fact that the algorithm for sorting editorial remarks and versions is identical seems like coincidental rather than fundamental duplication to me.